integrationTests/vm/arwen/erc20_test.go

```diff
@@ -0,0 +1,92 @@
+	context.executeSC(&context.Bob, "decreaseAllowance(address,uint256)@"+context.Alice.AddressHex()+"@"+formatHexNumber(5))
+
+	// Assertion
+	assert.Equal(t, uint64(42000), context.querySCInt("totalSupply()", [][]byte{}))
+	assert.Equal(t, uint64(1000), context.querySCInt("balanceOf(address)", [][]byte{context.Alice.Address}))
+	assert.Equal(t, uint64(1000), context.querySCInt("balanceOf(address)", [][]byte{context.Bob.Address}))
+	assert.Equal(t, uint64(495), context.querySCInt("allowance(address,address)", [][]byte{context.Alice.Address, context.Bob.Address}))
+	assert.Equal(t, uint64(495), context.querySCInt("allowance(address,address)", [][]byte{context.Bob.Address, context.Alice.Address}))
+
+	// Payments
+	context.executeSC(&context.Alice, "transferFrom(address,address,uint256)@"+context.Bob.AddressHex()+"@"+context.Carol.AddressHex()+"@"+formatHexNumber(50))
+	context.executeSC(&context.Bob, "transferFrom(address,address,uint256)@"+context.Alice.AddressHex()+"@"+context.Carol.AddressHex()+"@"+formatHexNumber(50))
+
+	// Assertion
+	assert.Equal(t, uint64(950), context.querySCInt("balanceOf(address)", [][]byte{context.Alice.Address}))
+	assert.Equal(t, uint64(950), context.querySCInt("balanceOf(address)", [][]byte{context.Bob.Address}))
+	assert.Equal(t, uint64(100), context.querySCInt("balanceOf(address)", [][]byte{context.Carol.Address}))
+}
+
+func Test_SOL_003(t *testing.T) {
+	context := setupTestContext(t)
+
+	context.deploySC("./testdata/erc20sol003/0-0-3.wasm", "")
+
+	// Minting
+	context.executeSC(&context.Owner, "transfer(address,uint256)@"+context.Alice.AddressHex()+"@"+formatHexNumber(1000))
+	context.executeSC(&context.Owner, "transfer(address,uint256)@"+context.Bob.AddressHex()+"@"+formatHexNumber(1000))
+
+	// Assertion
+	assert.Equal(t, uint64(1000), context.querySCInt("balanceOf(address)", [][]byte{context.Alice.Address}))
+	assert.Equal(t, uint64(1000), context.querySCInt("balanceOf(address)", [][]byte{context.Bob.Address}))
+
+	// Regular transfers
+	context.executeSC(&context.Alice, "transfer(address,uint256)@"+context.Bob.AddressHex()+"@"+formatHexNumber(200))
+	context.executeSC(&context.Bob, "transfer(address,uint256)@"+context.Alice.AddressHex()+"@"+formatHexNumber(400))
+
+	// Assertion
+	assert.Equal(t, uint64(1200), context.querySCInt("balanceOf(address)", [][]byte{context.Alice.Address}))
+	assert.Equal(t, uint64(800), context.querySCInt("balanceOf(address)", [][]byte{context.Bob.Address}))
+}
+
+func Test_C_001(t *testing.T) {
+	context := setupTestContext(t)
+
+	context.deploySC("./testdata/erc20c/wrc20_arwen.wasm", formatHexNumber(42000))
+
+	// Assertion
+	assert.Equal(t, uint64(42000), context.querySCInt("totalSupply", [][]byte{}))
+	assert.Equal(t, uint64(42000), context.querySCInt("balanceOf", [][]byte{context.Owner.Address}))
+
+	// Minting
+	context.executeSC(&context.Owner, "transferToken@"+context.Alice.AddressHex()+"@"+formatHexNumber(1000))
```

Comment:
use some constants

Reply:
Only if they increase test readability. Sometimes, additional constructs (such as extra constants in this case) might clutter the test.